src/main/java/com/rakovets/course/java/core/practice/lambda_expressions/MathStatistic.java

```diff
@@ -0,0 +1,41 @@
+package com.rakovets.course.java.core.practice.lambda_expressions;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class MathStatistic {
+    public static List<Integer> getRandomNumber (int numbersOfPrimes,int minValue,int maxValue) {
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"